docs/architecture/modernize-desktop/example-migration.md

```diff
@@ -18,57 +18,71 @@ The migration process consists of four sequential steps:
+Since the API surface in .NET Framework and .NET is similar but not identical, you must check which APIs are available on .NET and which aren't. You can use the .NET Portability Analyzer tool to surface APIs used that aren't present on .NET. It looks at the binary level of your app, extracts all the APIs that are called, and then lists which APIs aren't available on your target framework (.NET 5.0 in this case).
 
 You can find more information about this tool at:
 
 <https://docs.microsoft.com/dotnet/standard/analyzers/portability-analyzer>
 
-An interesting aspect of this tool is that it only surfaces the differences from your own code and not code from external packages, which you can't change. Remember you should have updated most of these packages to make them work with .NET Core.
+An interesting aspect of this tool is that it only surfaces the differences from your own code and not code from external packages, which you can't change. Remember you should have updated most of these packages to make them work with .NET.
 
 ### Migrate project file
 
-#### Create the new .NET Core project
+The [Try Convert](https://github.com/dotnet/try-convert/releases) tool is a great way to migrate a project. It's a global tool that attempts to upgrade your project file from the old style to the new SDK style, and retargets applicable projects to .NET 5. Once installed, you can run the following commands:
 
-In most of the cases, you'll want to update your existing project to the new .NET Core format. However, you can also create a new project while maintaining the old one. The main drawback from updating the old project is that you lose designer support, which may be important for you. If you want to keep using the designer, you must create a new .NET Core project in parallel with the old one and share assets. If you need to modify UI elements in the designer, you can switch to the old project to do that. And since assets are linked, they'll be updated in the .NET Core project as well.
+```dotnetcli
+try-convert -p "<path to your project file>"
+```
+
+Or:
+
+```dotnetcli
+try-convert -w "<path to your solution>"
+```
+
+After the tool attempts the conversion, reload your files in Visual Studio to run and test. There's a possibility that Try Convert won't be able to perform the conversion due to the specifics of your project. In that case, you can refer the below steps.
 
-The [SDK-style project](../../core/project-sdk/msbuild-props.md) for .NET Core is a lot simpler than .NET Framework's project format. And apart from the previously mentioned `PackageReference` entries, you won't need to do much more. The new project format includes certain file extensions [by default](../../core/tools/csproj.md#default-compilation-includes-in-net-core-projects), such as `.cs` and `.xaml` files, without the need to explicitly include them in the project file.
+#### Create the new .NET project
```

Comment:
```suggestion
#### Migrate manually
1. Create the new .NET project
```

Reply:
We should have 2 sections : migrate with the tool or manually so it is clear that if the tool worked, nothing else is needed. All sections bellow should be steps in manual migration, I'd use numbers to emphasize that.

Reply:
ok.